lte/gateway/python/integ_tests/s1aptests/test_attach_he_policy.py

```diff
@@ -0,0 +1,232 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+import time
+import ipaddress
+import gpp_types
+import s1ap_types
+import s1ap_wrapper
+import json
+from integ_tests.s1aptests.s1ap_utils import SessionManagerUtil, HeaderEnrichmentUtils
+from lte.protos.policydb_pb2 import FlowMatch, HeaderEnrichment
+
+
+class TestAttachServiceWithMultiPdnsAndBearers(unittest.TestCase):
```

Comment:
change the test name in line with the file name.

Reply:
what is the reason that you prefer to base the test on top of multi pdn and bearer testing?

Reply:
I changed base to test_attach_detach_rar_tcp_data.py.